modules/administration-guide/partials/proc_restoring-a-che-instance-from-a-backup-by-snapshot-id.adoc

```diff
@@ -0,0 +1,50 @@
+[id="restoring-a-{prod-id-short}-instance-from-a-backup-by-snapshot-id_{context}"]
+= Restoring a {prod-short} instance from a backup by snapshot ID
+
+To restore a {prod-short} instance from a particular backup snapshot, you can enter the snapshot ID with `{prod-cli}`.
+
+include::snip_warning-backup-snapshots-are-bound-to-specific-cluster.adoc[]
+
+IMPORTANT: If the backup snapshot was taken in an earlier version of Che, add the `--version=__<earlier_version_number>__` option to the `chectl server:restore` command to roll back Che to that earlier version.
+
+.Prerequisites
+
+* Installed `{prod-cli}`.
+* Backup snapshot of a {prod-short} instance on any of the following:
+** xref:backing-up-of-che-instances-to-an-sftp-backup-server.adoc[SFTP backup server]
+** xref:backing-up-of-che-instances-to-amazon-s3.adoc[Amazon S3 or S3 API compatible backup server]
+** xref:backing-up-of-che-instances-to-a-rest-backup-server.adoc[REST backup server]
+** xref:backing-up-of-che-instances-to-the-internal-backup-server.adoc[Internal backup server]
+* Backup snapshot ID, which can be found in:
+** Command-line output when creating a backup
+** `status` of a `CheClusterBackup` custom object
+** Command-line output of the restic command that can link:https://restic.readthedocs.io/en/stable/045_working_with_repos.html[list snapshots in the backup repository]
+
+.Procedure
+
+. Run the `{prod-cli} server:restore` command with the backup snapshot ID specified using the `--snapshot-id=` option:
++
+[subs="+quotes,+attributes"]
+----
+$ {prod-cli} server:restore \
+--snapshot-id=_<snapshot-id>_ \ <1>
+--repository-url=_<repository-url>_ \ <2>
+--repository-password=_<repository-password>_ <3>
+----
+<1> Specifies the snapshot ID.
```

Comment:
```suggestion
<1> The snapshot ID.
```
Here and in similar lists below: perhaps the items could all be noun-based for consistency?

Reply:
Thank you! Fixed.